docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts

```diff
@@ -0,0 +1,31 @@
+import * as _ from 'lodash'
+
+import { imports } from 'docs/src/components/Playground/renderConfig'
+import { ComponentSourceManagerLanguage } from 'docs/src/components/ComponentDoc/ComponentSourceManager'
+
+const name = 'stardust-ui-example'
+const description =
+  'An exported example from Stardust UI React, https://stardust-ui.github.io/react/'
+const dependencies = _.mapValues(imports, () => 'latest')
+const devDependencies = {
+  '@types/lodash': 'latest',
```

Comment:
why we have here types for `lodash`, but we don't have it inside the dependencies?

Reply:
These `devDependencies` are used for an exported example. I don't see any way to get them automatically like `dependencies`.

`@types/*` should be placed in `devDependencies` as CodeSandbox will check for them there.